WordPress/WordPressTest/NotificationFilteringServiceTests.swift

```diff
@@ -53,73 +53,44 @@ final class NotificationFilteringServiceTests: XCTestCase {
         XCTAssertTrue(sut.wordPressNotificationsEnabled)
     }
 
-    // MARK: - Disable WordPress notifications
+    // MARK: - Should disable WordPress notifications
 
-    func testDisableWPNotificationsIfNeededInJetpackWhenFeatureFlagEnabled() {
-        setup(allowDisablingWPNotifications: true, isWordPress: false)
-        XCTAssertTrue(sut.wordPressNotificationsEnabled)
-
-        sut.disableWordPressNotificationsIfNeeded()
-
-        XCTAssertFalse(sut.wordPressNotificationsEnabled)
-    }
-
-    func testDisableWPNotificationsIfNeededInJetpackWhenFeatureFlagDisabled() {
-        setup(allowDisablingWPNotifications: false, isWordPress: false)
-        XCTAssertTrue(sut.wordPressNotificationsEnabled)
-
-        sut.disableWordPressNotificationsIfNeeded()
-
-        XCTAssertTrue(sut.wordPressNotificationsEnabled)
-    }
-
-    func testDisableWPNotificationsIfNeededInWordPresskWhenFeatureFlagEnabled() {
-        setup(allowDisablingWPNotifications: true, isWordPress: true)
-        XCTAssertTrue(sut.wordPressNotificationsEnabled)
-
-        sut.disableWordPressNotificationsIfNeeded()
-
-        XCTAssertTrue(sut.wordPressNotificationsEnabled)
-    }
-
-    // MARK: - Should filter WordPress notifications
-
-    func testShouldFilterWordPressNotificationsInWordPressWhenFeatureFlagEnabledAndWordPressNotificationsDisabled() {
+    func testShouldDisableWordPressNotificationsInWordPressWhenFeatureFlagEnabledAndWordPressNotificationsDisabled() {
         setup(allowDisablingWPNotifications: true, isWordPress: true)
         sut.wordPressNotificationsEnabled = false
 
-        XCTAssertTrue(sut.shouldFilterWordPressNotifications())
+        XCTAssertTrue(sut.shouldDisableWordPressNotifications())
     }
 
-    func testShouldFilterWordPressNotificationsInWordPressWhenFeatureFlagDisabledAndWordPressNotificationsDisabled() {
+    func testShouldDisableWordPressNotificationsInWordPressWhenFeatureFlagDisabledAndWordPressNotificationsDisabled() {
         setup(allowDisablingWPNotifications: false, isWordPress: true)
         sut.wordPressNotificationsEnabled = false
 
-        XCTAssertFalse(sut.shouldFilterWordPressNotifications())
+        XCTAssertFalse(sut.shouldDisableWordPressNotifications())
     }
 
-    func testShouldFilterWordPressNotificationsInWordPressWhenFeatureFlagEnabledAndWordPressNotificationsEnabled() {
+    func testShouldDisableWordPressNotificationsInWordPressWhenFeatureFlagEnabledAndWordPressNotificationsEnabled() {
         setup(allowDisablingWPNotifications: true, isWordPress: true)
         sut.wordPressNotificationsEnabled = true
 
-        XCTAssertFalse(sut.shouldFilterWordPressNotifications())
+        XCTAssertFalse(sut.shouldDisableWordPressNotifications())
     }
 
-    func testShouldFilterWordPressNotificationsInJetpack() {
+    func testShouldDisableWordPressNotificationsInJetpack() {
```

Comment:
```suggestion
    func testShouldAllowNotificationsInJetpackWhenWordPressNotificationsAreDisabled() {
```
I think this test verifies that notifications in Jetpack are not inadvertently disabled when notifications in WordPress are disabled. If so, I would rename for clarity.

Reply:
👍 https://github.com/wordpress-mobile/WordPress-iOS/pull/19616#discussion_r1026117238